new_samples/worker/worker.go

```diff
@@ -0,0 +1,86 @@
+// Package worker implements a Cadence worker with basic configurations.
+package worker
+
+import (
+	"github.com/uber-common/cadence-samples/new_samples/workflows"
+	"github.com/uber-go/tally"
+	apiv1 "github.com/uber/cadence-idl/go/proto/api/v1"
+	"go.uber.org/cadence/.gen/go/cadence/workflowserviceclient"
+	"go.uber.org/cadence/activity"
+	"go.uber.org/cadence/compatibility"
+	"go.uber.org/cadence/worker"
+	"go.uber.org/cadence/workflow"
+	"go.uber.org/yarpc"
+	"go.uber.org/yarpc/transport/grpc"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+const (
+	HostPort = "127.0.0.1:7833"
+	Domain   = "cadence-samples"
+	// TaskListName identifies set of client workflows, activities, and workers.
+	// It could be your group or client or application name.
+	TaskListName   = "cadence-samples-worker"
```

Comment:
I think we can leave this for now, but for when we add / migrate more workflows:
should we perhaps make each sample use a different tasklist?

currently, if they all use the same tasklist (somewhat implied as the name is pretty generic), starting hello-world's worker will try to process tasks from other-sample's workflows, which it won't understand, making log-spam and failed decision tasks in history.

that's not, like, awful or anything.  just seems potentially worse than using unique tasklists.
unless the plan is to register everything in the one main.go?

Reply:
yes for the samples we would like to keep it simple and everything will be registered to a single binary. It is true that we may need to enable configuring tasklist in a more dynamic way but for samples we may just keep using one

Reply:
if it's all in the same binary (i.e. all workflows are registered all the time), one tasklist should be fine.  I just wasn't sure if it was going to follow the "every sample is fully self-contained" pattern like the other samples, or blend it all into one.

"blend it all into one" seems fine and fits with the folder/package structure, so yea 👍 all good I think.